frontend/src/components/inputs/Selector.vue

```diff
@@ -109,4 +109,17 @@ export default {
 .selector .vs__fade-leave-to {
   @apply rounded-xl opacity-0;
 }
+
+.selector.vs--disabled .vs__open-indicator,
+.selector.vs--disabled .vs__search {
+  @apply invisible;
+}
+
+.selector.vs--disabled .vs__selected {
+  @apply text-teal-light;
+}
+
+.selector.vs--disabled .vs__dropdown-toggle {
+  @apply bg-transparent border border-teal-light;
+}
```

Comment:
Does this conflict with #802? 🤔

Reply:
I'm not able to reproduce the bug fixed in #802 with my version.